scipy/special/_logsumexp.py

```diff
@@ -125,3 +125,29 @@ def logsumexp(a, axis=None, b=None, keepdims=False, return_sign=False):
         return out, sgn
     else:
         return out
+
+
+def softmax(x, axis=None):
+    r"""
+    Softmax function
+
+    Compute the softmax of each element along an axis of X.
+
+    .. math:: \sigma(X)_j = \frac{e^{x_j}}{\sum_{k=1}^{K}e^{x_k}} \textrm{ for } j = 1, \ldots, K
```

Comment:
Minor, optional: maybe simplify the formula for reading as plain text: move the `for j=...` clause to the text. Or even remove it entirely and remove the K variable and the summation limits (i.e., write simply `\sum_k ...`

Reply:
ack